activation/poet.go

```diff
@@ -465,6 +476,10 @@ func NewPoetServiceWithClient(
 	return service
 }
 
+func (c *poetService) GetTickSize() uint64 {
```

Comment:
NIT: `Get` as a prefix is unusual in Go.

```suggestion
func (c *poetService) TickSize() uint64 {
```

Reply:
It's pretty common, especially in autogenerated clients like:
```go
func (x *CheckpointStreamResponse) GetData() []byte {
        if x != nil {
                return x.Data
        }
        return nil
}
```
for the `Data` field in a struct.
but as you wish - i'm changing it.